### PR TITLE
ReshRateManager fixes and updates

### PR DESCRIPTION
**Pull request by danteruiz**
ReshRateManager fixes and updates

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19136/HighFidelity-Beta-PR15491-36b7acd.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/19136/HighFidelity-Quest-Beta-PR15491-36b7acd.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19136/HighFidelity-Beta-PR15491-36b7acd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19136/HighFidelity-Beta-Interface-PR15491-36b7acd.exe

File: interface/src/RefreshRateManager.cpp [hunk 1]
```diff
@@ -26,7 +26,7 @@ static const std::array<std::string, RefreshRateManager::RefreshRateProfile::PRO
     { { "Eco", "Interactive", "Realtime" } };
 
 static const std::array<std::string, RefreshRateManager::RefreshRateRegime::REGIME_NUM> REFRESH_RATE_REGIME_TO_STRING =
-    { { "Running", "Unfocus", "Minimized", "StartUp", "ShutDown" } };
+    { { "Running", "Unfocus", "Minimized", "StartUp", "ShutDown", "Inactive" } };
 
 static const std::array<std::string, RefreshRateManager::UXMode::UX_NUM> UX_MODE_TO_STRING =
     { { "Desktop", "HMD" } };
```
Comment (samcake) [inline interface/src/RefreshRateManager.cpp]: can you change HMD to VR ?

File: interface/src/RefreshRateManager.h [hunk 2]
```diff
@@ -33,6 +35,7 @@ class RefreshRateManager {
         MINIMIZED,
         STARTUP,
         SHUTDOWN,
+        INACTIVE,
```
Comment (samcake) [inline interface/src/RefreshRateManager.h]: I would suggest to put this inactive mode as number 2 between Running and Unfocused as it is the logical progression 
I would suggest to replace "running" by "Active_Focus"
and "inactive" by "Inactive_Focus"

File: interface/src/RefreshRateManager.cpp [hunk 3]
```diff
@@ -36,21 +36,20 @@ static const std::map<std::string, RefreshRateManager::RefreshRateProfile> REFRE
       { "Interactive", RefreshRateManager::RefreshRateProfile::INTERACTIVE },
       { "Realtime", RefreshRateManager::RefreshRateProfile::REALTIME } };
 
-static const std::array<int, RefreshRateManager::RefreshRateProfile::PROFILE_NUM> RUNNING_REGIME_PROFILES =
-    { { 5, 20, 60 } };
+static const std::array<int, RefreshRateManager::RefreshRateRegime::REGIME_NUM> ECO_PROFILE =
```
Comment (samcake) [inline interface/src/RefreshRateManager.cpp]: Can you put a comment that details each of the numbers in the profile
[ "regimeA", "regimeB... ]

**Comment by samcake** [comment]
build this please

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19153/HighFidelity-Beta-PR15491-36b7acd.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/19153/HighFidelity-Beta-Interface-PR15491-36b7acd.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19153/HighFidelity-Beta-PR15491-36b7acd.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/19153/HighFidelity-Beta-Interface-PR15491-36b7acd.exe

**Pull request closed without merging**